service/policy/db/schema_erd.md

```diff
@@ -122,7 +122,7 @@ erDiagram
         jsonb metadata "Metadata for the KAS (see protos for structure)"
         character_varying name UK "Optional common name of the KAS"
         jsonb public_key "Public key of the KAS (see protos for structure/options)"
-        integer source_type 
+        character_varying source_type 
```

Comment:
This was changed when running the make script, is it correct?

Reply:
Yeah, looks like the `models.go` has it as a `pgtype.Text` and IIRC, @c-r33d changed the column value from the proto enum index to the string representation